source/features/follow-file-renames.tsx

```diff
@@ -28,19 +28,22 @@ async function linkify(button: HTMLButtonElement, url: GitHubURL): Promise<void
 	const {files} = await getCommitInfo(sha.getAttribute('value')!);
 
 	for (const file of (files as File[])) {
-		if (file[fromKey] === url.filePath) {
+		if (file[fromKey] === filePath) {
 			if (file.status === 'renamed') {
-				url.assign({
+				const linkifiedURL = new GitHubURL(location.href);
+				linkifiedURL.assign({
 					route: 'commits',
 					filePath: file[toKey],
+					// Clear the search from the URL
```

Comment:
Don't think this comment is needed

Reply:
I thought the second part was obvious because the field was already inside of the link that was being passed to rename.

Reply:
The whole point of the comment is for the second part.

In other words "Why are we clearing the search".